be/src/common/config.h

```diff
@@ -331,6 +331,9 @@ CONF_mInt64(row_step_for_compaction_merge_log, "0");
 CONF_mInt32(base_compaction_trace_threshold, "10");
 CONF_mInt32(cumulative_compaction_trace_threshold, "2");
 
+// Threshold to logging agent task trace, in seconds.
+CONF_mInt32(agent_task_trace_threshold_sec, "2");
```

Comment:
Could you please update hte be_config.md document?

Reply:
Oh, this PR has been merged before I update docs, I'll do it in next PR.